modules/nodes-pods-autoscaling-custom-trigger.adoc

```diff
@@ -149,6 +152,9 @@ spec:
 
 You can scale pods based on an Apache Kafka topic or other services that support the Kafka protocol. The custom metrics autoscaler does not scale higher than the number of Kafka partitions, unless you set the `allowIdleConsumers` parameter to `true` in the scaled object or scaled job.
 
+:FeatureName: Autoscaling based on Apache Kafka metrics
+include::snippets/technology-preview.adoc[leveloffset=+0]
+
```

Comment:
I would consider some way of moving the note and TP warning around so that they aren't next to each other. (If possible - I know the peer review team has asked me to do this before, but sometimes it's not feasible.)